<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="palaiseau">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>680</y>
      <width>74</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Palaiseau ?</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDial" name="inputPower">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-30</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>-18</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPowerLCD">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>595</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="inputPowerLabel">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>540</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Power (dBm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="inputFrequencyLabel">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>400</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Frequency (MHz)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputFrequencyLCD">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>455</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="inputFrequency">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10</number>
    </property>
    <property name="maximum">
     <number>3000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>2410</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputVoltageLCD">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>315</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="inputVoltageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Targeted Voltage (mV)</string>
    </property>
   </widget>
   <widget class="QDial" name="inputVoltage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="rectifierCharacterization">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>210</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Rectifier characterization</string>
    </property>
   </widget>
   <widget class="QPushButton" name="timeCharging">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>160</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Time charging</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveFig">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>110</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Save figure</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="energyNumber">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="superCapacitorLabel">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>40</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Supercapacitor Energy (ÂµJ)</string>
    </property>
   </widget>
   <widget class="MplWidget" name="MplWidget" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>1050</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
